src/update/updater.js

```diff
@@ -168,40 +173,100 @@ class Updater {
 
       console.log('');
     });
+
+    await asyncForEach(this.wikiaLua, async (url) => {
+      const name = url.split('/')[url.split('/').length - 2].split(':')[1];
+      const filename = `${name}Data`;
+
+      console.log(`Fetching ${url}`);
+
+      let request = null;
+      let wikilua = null;
+
+      request = await fetch(url);
+      wikilua = await request.text();
+
+      const lines = wikilua.split('\n');
+      const modifiedLines = lines.slice(0, lines.length - 2).join('\n');
+
+      const lua = new Lua.LuaState();
+      lua.DoString(`
+        json = (loadfile "src/update/json.lua")()
+        ${modifiedLines}
+        file = io.open("cache/json/tmp_${filename}.json", "w")
+        io.output(file)
+        io.write(json.stringify(WeaponData))
+        io.close(file)
+      `);
+
+      const readFile = util.promisify(fs.readFile);
+      const json = await readFile(`${__dirname}/../../cache/json/tmp_${filename}.json`);
+
+      const ignore = JSON.parse(json).IgnoreInCount;
+      const unordered = JSON.parse(json).Weapons;
+      const ordered = {};
+
+      Object.keys(unordered).sort().forEach((key) => {
+        const unoderedInnerKeys = unordered[key];
+        const orderedInnerKeys = {};
+
+        Object.keys(unoderedInnerKeys).sort().forEach((innerKey) => {
```

Comment:
typo "unodered"

Reply:
Ah cheers, it was like 1am when I was reviewing this ;)